openrtb_ext/imp.go

```diff
@@ -84,3 +86,12 @@ type ExtStoredBidResponse struct {
 type Options struct {
 	EchoVideoAttrs bool `json:"echovideoattrs"`
 }
+
+// GetImpIDs returns slice of all impression Ids from impList
+func GetImpIDs(impList []openrtb2.Imp) []string {
```

Comment:
In many adapters it would be simpler to pass the request. Consider `GetImpIDsFromRequest(r openrtb.BidRequest)`

Reply:
But if we use GetImpIDsFromRequest(request), Do we need to consider whether it's necessary to pass the entire request data to the function, given that only the imp[].id contained in request.Imp is needed.

Reply:
I don't think there are any performance concerns with passing the entire request since it should be by reference. However, I don't have a strong opinion here. I'm ok with the current implementation as I tend to favor limiting what a function has access to.